ocpairprogramming/src/main/java/com/pino/project/ocpairprogramming/java8/ocp/chapter3/generics/Animal.java

```diff
@@ -0,0 +1,28 @@
+package com.pino.project.ocpairprogramming.java8.ocp.chapter3.generics;
+
+public class Animal implements Comparable<Animal> {
+	private int id;
+
+	public Animal(int id) {
+		super();
```

Comment:
why is the call to parent (Object) constructor needed?

Reply:
You're thouroghly right. Actually it was a default autogenerated by Eclipse when ask to 'Generate Constructor using fields'

Reply:
ma perché dovete parlare inglese se siamo solo noi italiani?